### PR TITLE
[C++/de-de] Fixed some typos in german translation

### PR DESCRIPTION
**Pull request by caminsha**
- [X] I solemnly swear that this is all original content of which I am the original author
- [X] Pull request title is prepended with `[language/lang-code]`
- [X] Pull request touches only one file (or a set of logically related files with similar changes made)
- [X] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [X] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [X] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: de-de/c++-de.html.markdown [hunk 1]
```diff
@@ -335,7 +336,7 @@ ECarTypes GetPreferredCarType()
 	return ECarTypes::Hatchback;
 }
 
-// Mit C++11 existiert eine einfache Möglichkeit einem Typ dem Enum zu zuweisen. Dies
+// Mit C++11 existiert eine einfache Möglichkeit einem Typ dem Enum zuzuweisen. Dies
```
Comment (Menelion) [inline de-de/c++-de.html.markdown]: Sind Sie sicher, _zuzuweisen_, nicht _zu zuweisen_?

File: de-de/c++-de.html.markdown [hunk 2]
```diff
@@ -818,7 +819,7 @@ void doSomethingWithAFile(const std::string& filename)
 // Container
 /////////////////////
 
-// Die Container der Standard template Bibliothek beinhaltet einige vordefinierter templates.
+// Die Container der Standard template Bibliothek beinhaltet einige vordefinierter Templates.
```
Comment (Menelion) [inline de-de/c++-de.html.markdown]: vordefinierte?

File: de-de/c++-de.html.markdown [hunk 3]
```diff
@@ -818,7 +819,7 @@ void doSomethingWithAFile(const std::string& filename)
...
 // Diese verwalten die Speicherbereiche für die eigenen Elemente und stellen Member-Funktionen
 // für den Zugriff und die Maniplulation bereit.
```
Comment (Menelion) [inline de-de/c++-de.html.markdown]: Manipulation

**Comment by Menelion** [comment]
@caminsha Could you please answer my questions in the comments and commit changes accordingly? These are really tiny, I'll merge the PR in then. Thanks!

File: de-de/c++-de.html.markdown (hunk 1 above)
Comment (caminsha) [reply de-de/c++-de.html.markdown]: Ich bin mir nicht 100% sicher, allerdings bin ich mir ziemlich sicher, dass es _zuzuweisen_ ist.

**Comment by caminsha** [comment]
@Menelion Thanks for your changes! I have changed two things which you mentioned. Thanks!

**Comment by Menelion** [comment]
@caminsha vielen Dank! :)

**Pull request merged**